src/components/GroupsTable/GroupsTable.cy.js

```diff
@@ -58,7 +58,8 @@ const checkSorting = (label, order, dataField) => {
   if (order === 'ascending') {
     cy.get(header).find('button').click();
   } else {
-    cy.get(header).find('button').click().click();
+    cy.get(header).find('button').click();
+    cy.get(header).find('button').click();
```

Comment:
`dblclick` does two clicks almost immediately, it doesn't really work here due to a slight delay in the table. So leaving it as just two separate `click` calls here